clients/admin-ui/src/features/datamap/reporting/DatamapReportTable.tsx

```diff
@@ -913,7 +949,7 @@ export const DatamapReportTable = () => {
     }
   };
 
-  if (isReportLoading || isLoadingHealthCheck) {
+  if (isReportLoading || isLoadingHealthCheck || isLoadingFidesLang) {
```

Comment:
Added is loading otherwise the table displays with the fides key and then switches to user friendly name.